msdsn/conn_str.go

```diff
@@ -203,14 +203,13 @@ func Parse(dsn string) (Config, map[string]string, error) {
 		var err error
 		p.EnableKerberos, err = strconv.ParseBool(enablekerberos)
 		if err != nil {
-			f := "invalid enablekerberos flag '%v': %v"
-			return p, params, fmt.Errorf(f, enablekerberos, err.Error())
+			return p, params, fmt.Errorf("invalid enablekerberos flag '%v': %v", enablekerberos, err.Error())
```

Comment:
The previous code was incorrect. Please correct to:
`return p, params, fmt.Errorf("invalid enablekerberos flag '%v': %w", enablekerberos, err)`